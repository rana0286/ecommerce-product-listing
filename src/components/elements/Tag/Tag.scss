$defaultColor: '#333333';
$red: '#cc3333';
$green: '#009900';
$white: '#ffffff';
$lightBlue: '#def1f4';

.tag {
  color: $white;
  display: inline-block;
  padding-top: 16px;

  &--exclusive {
    background-color: $green;
    padding-left: 35px;
    padding-right: 35px;
  }

  &--sale {
    background-color: $red;
    padding-left: 35px;
    padding-right: 35px;
  }
}
