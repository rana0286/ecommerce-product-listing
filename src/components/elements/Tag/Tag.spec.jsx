import React from 'react';
import { shallow } from 'enzyme';
import Tag from './Tag';
import renderer from 'react-test-renderer';

describe('src/components/elements/Tag', () => {
  it('Tag component should render', () => {
    const component = shallow(<Tag  tagType="sale" tagLabel="sale" />);
    const tree = renderer.
      create(<Tag tagType="exclusive" tagLabel="exclusive"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
    expect(component).toBeDefined();
  });
});
