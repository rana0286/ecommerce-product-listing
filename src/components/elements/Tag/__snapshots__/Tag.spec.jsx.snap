// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/Header Header component should render 1`] = `
<div
  className="tag--exclusive"
>
  exclusive
</div>
`;
